client2/src/game/gui/GUI.js

```diff
@@ -0,0 +1,535 @@
+// import InventoryBar from "./InventoryBar";
```

Comment:
This has already been converted to a component, file not needed. See the top level GUI component client2/src/components/game/gui/GUI.jsx

Reply:
Oh I see, now everything seems clearer to me. I thought the files in previous gui folder is still needed.
With all these files not being needed anymore.

This means that all files in `client/src/gui` folder will be converted to react component, am I correct? @Arcanorum

Reply:
Yes, everything in `client/src/gui` should be a React component, or just as part of another component if it is small and doesn't really need it's own one.

I have went through the rest of the GUI files and have updated the todo list https://github.com/Arcanorum/dungeonz/projects/5

For `GUI.js` a lot of the missing methods have been made redundant by being taken care of by moving relevant logic into the JSX template, but some things still do need moving over, mainly around drag and drop of items, but that will come with the inventory panel work.

Reply:
Removed at https://github.com/Arcanorum/dungeonz/pull/99/commits/559e46b0dab5d1a050e33f2b0faa246d76354a20

Reply:
> Yes, everything in `client/src/gui` should be a React component, or just as part of another component if it is small and doesn't really need it's own one.
> 
> I have went through the rest of the GUI files and have updated the todo list https://github.com/Arcanorum/dungeonz/projects/5
> 
> For `GUI.js` a lot of the missing methods have been made redundant by being taken care of by moving relevant logic into the JSX template, but some things still do need moving over, mainly around drag and drop of items, but that will come with the inventory panel work.

I removed it for now.
Will take a look the todo list and see if there's something *(unrelated to items)* that I can move over to GUI.jsx.